app/css/components/navigation-menu.css

```diff
@@ -11,40 +11,42 @@
   flex-grow: 1;
 
   text-align: center;
-  filter: brightness(5);
+
+  border-bottom: 0.3rem solid transparent;
   -webkit-filter: brightness(5);
-  border-bottom: .3rem solid transparent;
+  filter: brightness(5);
 }
 
 .navigation-menu__item-link {
   display: block;
 
   height: 4.75rem;
-  color: #fff;
   text-decoration: none;
-  font-size: 0;
-  background: url(../menu-icons/settings.svg) center 1.6rem no-repeat;
+  color: #fff;
+  background: url('../menu-icons/settings.svg') center 1.6rem no-repeat;
```

Comment:
I chose single quote to not have mental switch between js and css (and we had more single quotes in css than double quotes :)). Let me know if you disagree.